Sources/docc/DocCDocumentation.docc/adding-supplemental-content-to-a-documentation-catalog.md

```diff
@@ -62,7 +62,12 @@ habitat.
 ...
 ```
 
-To add an article to your documentation catalog, use a text editor and create a file with an appropriate title and add a `.md` extension.
+To add an article to your documentation catalog, use a text editor and create a
+file with an appropriate title and add an `.md` extension. DocC uses the file
+name you choose to determine the URL path string for the article, converting
+upper case letters to lower case, and converting spaces and punctuation to
+hyphens. DocC always uses the title from the level 1 header as the title of the
+article, and not the file name.
```

Comment:
This should also mention that article file names need to be unique because otherwise the multiple pages would have the same URL. I think this information is important enough to call out in an "Important" aside.

 ```suggestion
To add an article to your documentation catalog, use a text editor and create a file with an appropriate title and add an `.md` extension. 

> Important: 
> DocC uses the article's lowercased file name, with consecutive sequences of whitespace and punctuation replaced with a hyphen (`-`), as a path component in the URL for that page. 
> If two or more articles have the same file name, DocC will raise a warning and skip building documentation for all but one of those articles.
>
> DocC doesn't display the article file name in content. Instead it uses the title from the level 1 header in places where the article is referenced.
```

Reply:
Great idea. I started writing this information the other day, but ended up with the simpler paragraph. I think this works well, and the extra information is worth the space. Thanks.